6.x/base-how-to.md

```diff
@@ -413,3 +413,71 @@ Add whatever validation rules & inputs you want, in addition to name and passwor
 ```
 This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/ui/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs.
 
+<a name="enable-email-verification-in-backpack-routes"></a>
+### Enable email verification in Backpack routes
+
+In Backpack CRUD 6.2 we introduced the ability to require email verification when accessing Backpack routes.
+
+To enable this feature please do the following: 
+
+**Step 1** - Ensure your Authenticable model (usually `App\Models\User`) implements the `Illuminate\Contracts\Auth\MustVerifyEmail` contract.
+
+```php
+<?php
+ 
+namespace App\Models;
+ 
+use Illuminate\Contracts\Auth\MustVerifyEmail;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Notifications\Notifiable;
+ 
+class User extends Authenticatable implements MustVerifyEmail
+{
+    use Notifiable;
+ 
+    // ...
+}
+```
+More info in: [Preparing model for email verification](https://laravel.com/docs/10.x/verification#model-preparation)
+
+**Step 2** - Make sure your Authenticable model table has a `email_verified_at`timestamp column.
+
+New Laravel instalations already bring this column, but if you came from earlier versions it's possible that `email_verified_at` column is missing. 
+
+You can quickly create a new migration that add that column with the following code: (please adjust table name if different from default). 
```

Comment:
These are not three ideas, but one. So it doesn't need three paragraphs, but one. Prettier but also more readable.

```suggestion
**Step 2** - Make sure your user model table has a `email_verified_at` column (timestamp). New Laravel installations already have it, but if you upgraded from L8/L9 it's possible that the `email_verified_at` column is missing. You can quickly create a new migration to add that column (example below) then run `php artisan migrate` to run it. [More info(https://laravel.com/docs/10.x/verification#database-preparation).
```

Reply:
I look at your text now and I only see: 
![image](https://github.com/Laravel-Backpack/docs/assets/7188159/ff58d8e1-a7e7-4caa-958f-d3b6c5b823f6)

It can be improved I agree, with your wording more than mine, but maybe separate the "fix" for those who need it ?f
```suggestion
**Step 2** - Make sure your user model table has a `email_verified_at` column (timestamp). New Laravel installations already have it, but if you upgraded from L8/L9 it's possible that the `email_verified_at` column is missing.

You can quickly create a new migration using `php artisan make:migration add_email_verified_at_column_to_users --table=users`, use the code below, then run `php artisan migrate`. [More info](https://laravel.com/docs/10.x/verification#database-preparation).
```
I still don't like that I have to read the code below, then remember that I need to run artisan migrate. Probably will re-read.

While if they are in order, you just follow:
- use the code below
**code**
- run migrate
- advance step

But I agree with "page flow", not sure it's the best for a step by step tutorial (basically it's what this is).

Reply:
Hmm good point. That is a separate idea here, and it's long, so a new paragraph is needed. Plus that sentence is waaay to long and complex, so maybe we split it up into multiple sentences - easier to read that way.

How about this?

```suggestion
**Step 2** - Make sure your user model table has a `email_verified_at` column (timestamp). New Laravel installations already have it, but if you upgraded from L8/L9 it's possible that the `email_verified_at` column is missing.

You can quickly create a new migration using `php artisan make:migration add_email_verified_at_column_to_users --table=users`, then use the code below:
```